db/sqlinterfaces.c

```diff
@@ -7862,7 +7862,8 @@ int handle_newsql_requests(struct thr_handle *thr_self, SBUF2 *sb,
             strcasecmp(sql_query->dbname, thedb->envname)) {
             logmsg(LOGMSG_ERROR, "DB name mismatch query:'%s' actual:'%s' \n",
```

Comment:
In that case, now you can do logmsg(LOGMSG_ERROR, errstr);
(may need to add \n at the end of errstr)